index.html

```diff
@@ -0,0 +1,187 @@
+            <li><a>Css</a></li>
+            <li><a>JavaScript</a></li>
+            <li><a>html</a></li>
+          </ul>
+          <div class="btn-more"><a href="#">Learn More</a></div>
+        </section>
+
+      </article>
+
+      <article class="card flex column width100">
+
+        <div class="card-image width100 height50"></div>
+
+        <section class="img-info flex column height50">
+          <h2>Multi-Post Stories<br>Gain+Glory</h2>
+          <ul class="flex ul-skills">
+            <li><a>Ruby on Rails</a></li>
+            <li><a>Css</a></li>
+            <li><a>JavaScript</a></li>
+            <li><a>html</a></li>
+          </ul>
+          <div class="btn-more"><a href="#">Learn More</a></div>
+        </section>
+
+      </article>
+
+      <article class="card flex column width100">
+
+        <div class="card-image width100 height50"></div>
+
+        <section class="img-info flex column height50">
+          <h2>Multi-Post Stories<br>Gain+Glory</h2>
+          <ul class="flex ul-skills">
+            <li><a>Ruby on Rails</a></li>
+            <li><a>Css</a></li>
+            <li><a>JavaScript</a></li>
+            <li><a>html</a></li>
+          </ul>
+          <div class="btn-more"><a href="#">Learn More</a></div>
+        </section>
+
+      </article>
+
+      <article class="card flex column width100">
+
+        <div class="card-image width100 height50"></div>
+
+        <section class="img-info flex column height50">
+          <h2>Multi-Post Stories<br>Gain+Glory</h2>
+          <ul class="flex ul-skills">
+            <li><a>Ruby on Rails</a></li>
+            <li><a>Css</a></li>
+            <li><a>JavaScript</a></li>
+            <li><a>html</a></li>
+          </ul>
+          <div class="btn-more"><a href="#">Learn More</a></div>
+        </section>
+
+      </article>
+
+      <article class="card flex column width100">
+
+        <div class="card-image width100 height50"></div>
+
+        <section class="img-info flex column height50">
+          <h2>Multi-Post Stories<br>Gain+Glory</h2>
+          <ul class="flex ul-skills">
+            <li><a>Ruby on Rails</a></li>
+            <li><a>Css</a></li>
+            <li><a>JavaScript</a></li>
+            <li><a>html</a></li>
+          </ul>
+          <div class="btn-more"><a href="#">Learn More</a></div>
+        </section>
+
+      </article>
+
+
+
+    </section>
+
+    <section class="about-me">
+      <h1 class="pd-tp-bt">
+        About me
+      </h1>
+      <p class="pd-bt">
+        I can help you build a product , feature or website Look through some of my work and experience! If you like
+        what you
+        see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <div class="btn-more btn-resume"><a href="#">Get My Resume</a></div>
+
+      <div class="about-img-1 abs wh-100 bg-contain"></div>
+      <div class="about-img-2 wh-50 abs bg-contain"></div>
+    </section>
```

Comment:
- [ ] Here you missed a whole section. Please add that section. 
<img width="282" alt="Capture" src="https://user-images.githubusercontent.com/58504655/120822959-59e32700-c570-11eb-96f3-e99ce7dd658c.PNG">